src/stories/story-utils/storeSetup.ts

```diff
@@ -4,15 +4,17 @@ import {namespace} from 'vuex-class'
 const flowVuexNamespace = namespace('flow')
 const builderVuexNamespace = namespace('builder')
 import {findFlowWith, IBlock, IContext, IFlow} from '@floip/flow-runner'
-import {get} from 'lodash'
+import {get, isEmpty, cloneDeep} from 'lodash'
+
+let storyInitState: any = {}
 
 /**
  * Safe register block module
  * Because some weird race condition is leading to modules not getting unregistered when clicking between stories before the next story re-registers
  */
 export const safeRegisterBlockModule = async function(BLOCK_TYPE: string, blockTypeStore: IRootState) {
   // @ts-ignore - TS2551: Property '$store' does not exist on type
-  if (this.$store.state['flow'][BLOCK_TYPE]) {
+  if (this.$store.hasModule(['flow', BLOCK_TYPE])) {
```

Comment:
this is the correct way to check if a module has been registered, and avoids us having this issue `[vuex] trying to unregister module '....', which is not registered`